src/ethereum/shanghai/fork.py

```diff
@@ -66,6 +67,7 @@
 ELASTICITY_MULTIPLIER = 2
 GAS_LIMIT_ADJUSTMENT_FACTOR = 1024
 GAS_LIMIT_MINIMUM = 5000
+INCLUSION_LIST_GAS = 4194304 # 2^22 gas for the IL (to start)
```

Comment:
I don't think we should make this a constant, this should be dynamic.

Reply:
this is the maximum. there needs to be a max!